build/dnsmasq/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:edge
```

Comment:
Ideally this should not be part of this repo but available as a global image on the Liip docker registry.

Reply:
Maybe in a dedicated repo so it can be managed in parallel of Pontsun. But this can be later of course.